core/aws-core/src/main/java/software/amazon/awssdk/awscore/retry/AwsRetryStrategy.java

```diff
@@ -83,7 +83,7 @@ private AwsRetryStrategy() {
      *
      * @return A retry strategy that does not retry.
      */
-    public static RetryStrategy<?, ?> none() {
+    public static RetryStrategy none() {
```

Comment:
doNotRetry, to match doNotBackoff?

Reply:
Sounds good, but, do we have a doNotBackoff defined anywhere? For backoff strategy the corresponding is called `retryImmediately` not `doNotBackoff`